models/base/manifest/snowplow_web_base_quarantined_sessions.sql

```diff
@@ -20,7 +20,7 @@ This significantly reduces table scans
 
 with prep as (
   select
-    cast(null as {{ snowplow_utils.type_max_string() }}) session_id
+    cast(null as {% if target.type == 'redshift' %} varchar(400) {% else %} {{snowplow_utils.type_max_string()}} {% endif %}) session_id
```

Comment:
I think it is safe to leave at that. Based on [this](https://snowplow.slack.com/archives/CKQETG3QX/p1620380872395400) and the PII enrichment [schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/2-0-0
) the max it can be is 128 characters.